Documentation/compatibility/wpf - AppDomain shutdown handling may now invoke in cleanup of WeakEvents.md

```diff
@@ -0,0 +1,46 @@
+## WPF AppDomain Shutdown Handling May Now Invoke in Cleanup of Weak Events
```

Comment:
Invoke is an active verb; it requires a direct object -- the thing that is invoked. Do you mean to say "...may now invoke a Dispatcher from the Finalizer thread during Cleanup of Weak Events"?

Reply:
It should probably say "invoke on the associated Dispatcher" as invoking is a concept within WPF where we queue a delegate to a dispatcher for immediate or later execution.

Reply:
Ended up going with "Calls Dispatcher.Invoke" (with appropriate tags) as its a bit more on the nose.